### ISSUE TITLE
define grammatical structure

### ISSUE THREAD

**Issue by masterkrang** [masterkrang/kako#10]
this is not going to be easy, but we need to define the grammatical structure for Kako.

**Comment by masterkrang** [comment]
I may read this book to help with this adventure. http://www.amazon.com/The-Atoms-Language-Hidden-Grammar/dp/0465005225

**Comment by JoseUgalde** [comment]
Not easy, but probably one of the most interesting parts.
First things that comes to my mind... 
(a) Isolating or Synthetic (if Synthetic, three possibilities: fusional, agglutinative or polysynthetic).
(b) What are our (I don't know the English term)... "partes del discurso", this is... our categories of words like (adjectives, nouns, pronouns, adverbs, verbs, prepositions, etc.)
(c) Subject/Object/Verb position.
There are three possibilities:1. SVO as in English: "I am a student".2. SOV as in Latim: "studiatis sum"3. OVS (I don't know examples of this one, but I know there are languages with this structure).

(d) Ergative/Absolutive resolution (this one is hard, I'm gonna have to research it).

Date: Mon, 19 Oct 2015 20:00:20 -0700

**Comment by masterkrang** [comment]
@JoseUgalde yes, exactly. So are you sure that these are the only 3 grammatical structures we could use? SVO, OVS, and SOV? If that is the case, I believe we could easily determine which structure to use.

More info: https://www.quora.com/What-are-the-most-common-grammatical-structures-for-sentences-in-different-languages

http://lingwiki.com/index.php?title=List_of_SVO_Languages
http://lingwiki.com/index.php?title=List_of_SOV_Languages

http://wals.info/

**Comment by masterkrang** [comment]
@JoseUgalde the "partes del discurso" are called "parts of speech" in English. FYI. I have created a new issue where we can discuss parts of speech: https://github.com/masterkrang/kako/issues/14

http://www.aims.edu/student/online-writing-lab/grammar/parts-of-speech

**Comment by masterkrang** [comment]
@JoseUgalde also created a separate thread for Isolating / Sythetic, which I don't understand but your insights are welcome https://github.com/masterkrang/kako/issues/15
